moduleframework/common.py

```diff
@@ -457,10 +461,31 @@ def get_url(self):
         """
         get location of repo(s) or image
 
-        :return:
+        :return: str
         """
         return self.info.get("url")
 
+    def get_template(self):
+        """
+        get location of template from config.yaml file
+        :return: str
+        """
+        return self.info.get("template")
+
+    def get_docker_pull(self):
+        """
+        Gets boolean value if image should be pulled or not.
+        :return: bool
+        """
+        docker_pull = self.info.get("docker_pull")
+        print_debug(docker_pull)
```

Comment:
this will just print "true", or "0" right? Would be beneficial to prepend a message what the value represents.

Reply:
This is typo. It will be deleted.